### PR TITLE
Update - Test API responses

### PR DESCRIPTION
**Pull request by joaherrerama**
see title

### CODE REVIEW

File: actinia_stac_plugin/core/stac_instance_id.py [hunk 1]
```diff
@@ -48,12 +50,8 @@ def deleteStacInstance(stac_instance_id):
         del instances[stac_instance_id]
         redis_actinia_interface.update("stac_instances", instances)
     except Exception:
-        return {
-            "Error": "Something went wrong please that the element is well typed "
+        raise BadRequest(
+            "Something went wrong please that the element is well typed "
```
Comment (mmacata) [inline actinia_stac_plugin/core/stac_instance_id.py:54]:
```suggestion
            "Something went wrong. Please check that the element is well typed "
```

**Review by mmacata — APPROVED** [review]
Yei - more tests! Thanks. 
Did you let them run locally? Sometimes the order is not as expected.
I only found a typo, else please merge

**Comment by joaherrerama** [comment]
I ran it locally, but I would create a workflow to void any break :)

**Comment by joaherrerama** [comment]
I was not the more elegant solution. But the ordering is done. It is important to empty the `redis-db` before running the test code

**Review by mmacata — APPROVED** [review]

**Pull request merged**